packages/discord.js/src/structures/interfaces/Collector.js

```diff
@@ -58,6 +58,12 @@ class Collector extends EventEmitter {
      */
     this.collected = new Collection();
 
+    /**
+     * The items that were not collected by this collector
+     * @type {Collection}
+     */
+    this.ignored = new Collection();
```

Comment:
are there any reason why r u storing it in a collection?
bc I don't see any use case here like `collected` collection have which is used in `end` event

Reply:
For the exact same reason there is a collected collection. It's useless to me, but if someone added it there must be a use, and you may always want to do something with the ignored items at the end, who knows. Either way, the possibility is there

Reply:
Well, then I'll prefer adding it in the end event then. `collected, ignored, reason`.

Reply:
JFYI, I'm not approving unless this is removed.

This collection is useless, it can and **will** result on extremely large memory usage, for something not a single soul cares about. Entries that are ignored should stay ignored, no mirroring from the collected items, which are something we care about.

Reply:
You're right, I didn't think of that